llvm/lib/Target/RISCV/RISCVTargetMachine.cpp

```diff
@@ -278,21 +278,7 @@ class RVVRegisterRegAlloc : public RegisterRegAllocBase<RVVRegisterRegAlloc> {
 
 static bool onlyAllocateRVVReg(const TargetRegisterInfo &TRI,
```

Comment:
Was TRI an unused argument before?

Reply:
Yes, it has prototype `llvm/include/llvm/CodeGen/RegAllocCommon.h::RegClassFilterFunc`.